packages/api/tests/endpoints/test-granules.js

```diff
@@ -553,10 +719,54 @@ test.serial('DELETE deleting an existing granule that is published will fail', a
     message,
     'You cannot delete a granule that is published to CMR. Remove it from CMR first'
   );
+
+  // granule should still exist in Dynamo and PG
+  t.true(await granulePgModel.exists(t.context.knex, { granule_id: granuleId }));
+  t.true(await granuleModel.exists({ granuleId }));
+
+  // Verify files still exist in S3 and PG
+  await Promise.all(
+    newGranule.files.map(async (file) => {
+      t.true(await s3ObjectExists({ Bucket: file.bucket, Key: file.key }));
+      t.true(await filePgModel.exists(t.context.knex, { bucket: file.bucket, key: file.key }));
+    })
+  );
 });
 
-test('DELETE deleting an existing unpublished granule', async (t) => {
-  const buckets = {
+test.serial('DELETE deleting an existing unpublished granule', async (t) => {
+  const newGranule = await createGranuleAndFiles(
+    t.context.knex,
+    t.context.collectionCumulusId,
+    false
+  );
+
+  const response = await request(app)
+    .delete(`/granules/${newGranule.granuleId}`)
+    .set('Accept', 'application/json')
+    .set('Authorization', `Bearer ${jwtAuthToken}`)
+    .expect(200);
+
+  t.is(response.status, 200);
+  const { detail } = response.body;
+  t.is(detail, 'Record deleted');
+
+  // verify the files are deleted from S3 and Postgres
+  await Promise.all(
```

Comment:
Sorry if I'm being dense: but do we not need to verify that the granule was deleted from dynamo/PG?

Reply:
Ah, is it because there are separate tests for the delete handler and so if the endpoint returns 200 we can assume that everything worked?

Reply:
Hmm this seems like an oversight. I think I added the checks for PG/S3 files in one of the recent changes. The endpoint tests here should probably either check that both Granules and Files are deleted or not check any of it because it's not _really_ the responsibility of the `del()` handler.

I think I'm going to say that these endpoint tests should be sorta like integration tests, looking at the full delete flow and making sure that granules and files are deleted:

16c38016480f243bde3a74b3aa9d86bdb0c625d4